src/build.sh

```diff
@@ -154,6 +161,16 @@ function kernel_build()
   return "$ret"
 }
 
+# This function adds the build clean option
+# integrated to env in order to clean the kernel.
+function build_clean()
+{
+  local flag="$1"
+
+  command="make clean${flag}"
+  cmd_manager "$command"
```

Comment:
Search for the use of `cmd_manager` in the code, and you will notice that this function expects a `flag` as its first parameter.